addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -290,7 +292,7 @@ export const dateField = {
             help: _t(`Displays a warning icon if the input dates are in the future.`),
         },
     ],
-    supportedTypes: ["date"],
+    supportedTypes: ["date", "datetime"],
```

Comment:
I would remove this and only keep the option, because
 - we don't need two ways to achieve a single goal
 - with the option, the widget remains datetime, so we have the correct column widths in list views (if we would set a date widget, the column would be narrower, and we would not have enough place to display the time when the input would be focused)